### PR TITLE
issue253 Use extend with overlap with retile in vectorize

### PR DESCRIPTION
**Pull request by ElienVandermaesenVITO**
#253

### CODE REVIEW

File: openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcesses.scala [hunk 1]
```diff
@@ -617,13 +617,19 @@ class OpenEOProcesses extends Serializable {
     val maxExtent = datacube.metadata.extent
     //naive approach: combine tiles and hope that we don't exceed the max size
     //if we exceed the max, vectorize will run on separate tiles, and we'll need to merge results
-    val newCols = Math.min(256*20,layout.cols)
-    val newRows = Math.min(256*20,layout.rows)
+    val newCols = Math.min(256,layout.cols)
```
Comment (jdries) [inline openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcesses.scala]: Here I suggest to still work with a larger tile size by default, maybe 256*4, avoiding that we make an immediate change from very coarse tiling to small tiles.
It is always possible that certain edge cases were not detected in our testing, and by doing it like this we do a more gradual rollout.
It is however important to ensure that the unit test also continues to work on the more complex case. Making this tile size configurable would be one way to achieve that.

**Review by jdries — CHANGES_REQUESTED** [review]

**Comment by jdries** [comment]
@ElienVandermaesenVITO I'm considering to merge this one, but is the remaining issue described or logged somewhere?
